repair/row_level.cc

```diff
@@ -2523,7 +2534,7 @@ class row_level_repair {
             auto schema_version = s->version();
             bool table_dropped = false;
 
-            repair_meta master(_ri.db,
+            auto rm = make_lw_shared<repair_meta>(_ri.db,
```

Comment:
Why this change? You could keep it as it is, and register a pointer to `master` in the map.

Reply:
I wanted the _repair_metas_masters  and _repair_metas (existing) one to be the same.

Reply:
Using row pointer is even better in turns of help debugging.